policy/modules/roles/staff.te

```diff
@@ -23,10 +23,18 @@ optional_policy(`
 	auditadm_role_change(staff_r)
 ')
 
+optional_policy(`
+	container_user_role(staff, staff_t, staff_application_exec_domain, staff_r)
+')
+
 optional_policy(`
 	dbadm_role_change(staff_r)
 ')
 
+optional_policy(`
+	docker_user_role(staff, staff_t, staff_application_exec_domain, staff_r)
+')
+
```

Comment:
What are your thoughts on putting `docker_user_role` and `podman_user_role` calls into `container_user_role`? It seems right.

Reply:
No containers for `user_t`?

Reply:
> What are your thoughts on putting `docker_user_role` and `podman_user_role` calls into `container_user_role`? It seems right.

I agree with this change.

> No containers for `user_t`?

I opted not to add the access for `user_t` because in my opinion containers (even if unprivileged) are mostly a privileged function. I think a tunable would be OK though.

Reply:
We can keep it as-is for now. It's easy enough to add later if needed.